Assets/LoggingManager/LoggingManager.cs

```diff
@@ -47,6 +30,10 @@ public class LoggingManager : MonoBehaviour
     [SerializeField]
     private bool enableCSVSave = true;
 
+    [Header("Log string Settings")]
+    [SerializeField]
+    private bool logStringOverTime = true;
+
```

Comment:
I think a better header could be "Logging Mode".
Add a underneath `Tooltip` explaining the potential performance gains / trade off between setting this on or off.